metallb/overlays/moc/zero/kustomization.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: metallb-system
+
+resources:
+  - ../../../base
+
+configMapGenerator:
+  - name: config
+    files:
+      - config=config.yaml
```

Comment:
Are you generating a configmap from a file which is already a configmap? Why don't you use the `configmap.yaml` directly as a resource?

Reply:
I was, but not intentionally. Thanks for catching that.